src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -842,6 +843,19 @@ export abstract class _MatAutocompleteTriggerBase
     // but the behvior isn't exactly the same and it ends up breaking some internal tests.
     overlayRef.outsidePointerEvents().subscribe();
   }
+
+  /** This stopPropagation is needed to openPanel works from outside through matAutocompleteTrigger. */
+  private _preventPropagationClickFromOutside(): void {
+    if (!this.panelOpen) {
+      fromEvent(this._document, 'click')
+        .pipe(take(1))
+        .subscribe((event: any) => {
+          if (this._element.nativeElement !== event.srcElement && event.isTrusted) {
+            event.stopImmediatePropagation();
```

Comment:
I don't think that we should be doing this, because it could prevent other unrelated click events from happening. The user can call `event.preventDefault` if this is a problem for them. 

There's also the problem that this will leak event listeners if the panel isn't opened through a click event.